Azure.ResourceManager.Core/ResourceGroupOperations.cs

```diff
@@ -64,6 +113,12 @@ public async override Task<ArmResponse<ResourceGroup>> GetAsync(CancellationToke
             });
         }
 
+        /// <summary>
```

Comment:
update to follow this
https://github.com/m-nash/azure-proto-sdk/blob/master/XmlDocStandard.md#tags